CarpetCleaning/StoredProcedures/Dbo.Loadexceldata.sql

```diff
@@ -0,0 +1,93 @@
+CREATE PROCEDURE DBO.JobDetails_LoadExcelData
+AS
+BEGIN
+
+WHILE ((SELECT COUNT(*) FROM ExcelData WHERE LoadDataNotes = 'Passed Validation') > 0)
```

Comment:
SELECT ... WHERE LoadDataNotes = 'Passed validation' - so if we get an error and have to rollback the transaction, then we just mark this row's LoadDataNotes with something other than Passed validation and it won't get picked up again in this loop, which is what we want otherwise this while becomes infinite.

Reply:
I don't understand what I need to do with that.